src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -168,3 +168,17 @@ def _list_log_files_with_filter(  # pylint: disable=too-many-arguments
         files.append(f)
 
     return files
+
+
+#        Custom funcs for list servers        #
+def _server_list_custom_func(
```

Comment:
nit: `_` (underscore) not required here as the function is exposed publicly as it's used by `commands.py`